### PR TITLE
NEXUS-9274 Updating NPM section with Scope and Login details

### PR DESCRIPTION
**Pull request by joedragons**
* Updated with https://issues.sonatype.org/browse/NEXUS-8332 details
* Updated with https://issues.sonatype.org/browse/NEXUS-8333 details
* Minor word changes
* Minor line/spacing changes (on areas I adjusted only)
* Typo fix that was causing view in github UI to be messed up (seems ok in live doc)

Brain fart, sorry for bad branch name.

### CODE REVIEW

File: chapter-npm.asciidoc [hunk 1]
```diff
@@ -159,9 +159,20 @@ Alternately, you can edit your package.json file and add a +publishConfig+ secti
...
+line for authentication of this combination will be automatically added to your +.npmrc+.  Nexus supports 
+specifically targeted registries and scoped packages using +npm login+ as well, otherwise the authentication will 
+be used for all registries and packages unless further authentication definitions are configured.  Details on 
++npm login+ can be seen on the https://docs.npmjs.com/cli/adduser[npm website].
+
+Usage of authentication in this manner further requires the +npm Bearer Token Realm+ to be activated. To do this, 
+simply add the realm to the active realms in the +Realms+ feature of the +Security+ menu from the 
++Administration+ menu in Nexus.
+
+Alternatively, you can authenticate by manually adding an +_auth+ value to +.npmrc+. The value has to be
```
Comment (cstamas) [inline chapter-npm.asciidoc]: We would like to emphasize that "manual" setup is discouraged.

Comment (joedragons) [inline chapter-npm.asciidoc]: @cstamas The ultimate way to discourage would be to remove this section.  Do you think there's value to keeping it?  I was thinking about something like "In the event you cannot use +npm login+, you can authenticate manually, however we advise caution as using manual additions as well as +npm login+ might cause conflicts within your .npmrc.  To authenticate manually add..."

Then I wondered why we have it at all.  Thoughts welcome (when you return is fine=)).

Comment (cstamas) [inline chapter-npm.asciidoc]: Am fine with removing it.

**Comment by cstamas** [comment]
+1

**Comment by joedragons** [comment]
Squash merge e4b17c7 with nexus-3.0.x.

**Pull request closed without merging**